### ISSUE TITLE
Keeping Sprites on stage?

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap#139]
Is there an option anywhere to force a sprite to stay within the stage area?

**Comment by bromagosa** [comment]
Isn't this what `[if on edge, bounce]` does, when combined with the appropriate rotation style?

**Comment by jmoenig** [comment]
I think this is about "modes". Scratch has a "fence" mode (well, Scratch *only* has that mode) that keeps sprites onstage always.

**Comment by brianharvey** [comment]
Yeah, I've wanted this (and wrap mode!) since forever.  But it's not a 5.0 showstopper, so let's just leave it open for now.
